packages/server/src/models/StreamSplitter.ts

```diff
@@ -106,7 +108,7 @@ export class StreamSplitter extends stream.Writable {
 
     await this.fileHandle.close()
 
-    this.emit('chunkFinished', {
+    await this.emitEvent('chunkFinished', {
```

Comment:
Can you explain why this change was necessary?

Reply:
to keep the events interface consistent